3 - PB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1796,6 +1796,33 @@ Questi ultimi dovranno agire di conseguenza risolvendo i problemi emersi o piani
 
 Per avere traccia degli elementi verificati che costituiscono il prodotto, su GitHub è possibile visualizzare l'insieme delle pull request approvate ed integrate mediante merge.
 
+== Processo di Validazione <processo_validazione>
+
+_Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.4.11_
+
+=== Scopo
+
+Il processo di Validazione serve a fornire prove oggettive che il sistema, quando in uso, soddisfi i requisiti esposti dal Proponente descritti nell'#adr.
```

Comment:
quel quando in uso nom aggiunge niente secondo me al discorso (cioè è ovvio che il programma chiuso non funziona imho) io lo toglierei